share/functions/__fish_print_ninja_targets.fish

```diff
@@ -0,0 +1,5 @@
+function __fish_print_ninja_targets
+    if [ -f build.ninja ]
+        ninja -t targets | sed 's/:.*//'
```

Comment:
Also, we try to use our `string` tool in our completions - `string replace -r ':.*' ''`.